scripts/valhalla_build_elevation

```diff
@@ -235,6 +294,19 @@ if __name__ == "__main__":
     elif args.verbosity >= 2:
         LOGGER.setLevel(logging.DEBUG)
 
+    config = None
+    if args.config:
+        with open(args.config) as f:
+            config = json.load(f)
+
+    if args.outdir:
+        elevation_fp = args.outdir
+    elif config is not None:
+        elevation_fp = Path(config["additional_data"]["elevation"] or "elevation")
```

Comment:
this is not quite enough to make `config` optional as it's also used further down when the user wants to get tiles covering the graph (often the most sensible option IMO in case elevation is only used to enrich the graph, not the `/height` service). admittedly there was no exception handling in place before either, `config` wasn't even `required`. 

guess it's enough to protect line 315 from missing config path.

Reply:
I guess I should point out the motivation for touching this as well. While the normal case would have a Valhalla JSON config, I think it's generally useful to have an option to run this script without that context, such as to download the whole planet elevation data to an internal file server. This is probably a minority use case, but I think it's useful to be able to operate as the previous shell script without a JSON config.

That said, good call. I neglected the possible code path on line 315. Any objection to checking that config is not none, exiting if it's not, and updating docs? This might be possible in argparse, but I don't know a great way off the top of my head.

Reply:
yeah I figured so, should've commented that I thought it's a valid use case, now the script can be used independently of valhalla. just get it from github's raw endpoint and execute it to get some bbox's tilezen tiles, that's great. would you mind adding some notes about that to the PR description for future reference?

> Any objection to checking that config is not none, exiting if it's not, and updating docs

that's what I had in mind too:)

Reply:
Sure thing; I'll update the PR description.